src/current/v25.1/monitor-and-analyze-transaction-contention.md

```diff
@@ -0,0 +1,592 @@
+---
+title: Monitor and Analyze Transaction Contention
+summary: How to monitor transaction contention using console, metrics, and crdb_internal tables. How to analyze transaction contention using crdb_internal tables.
+toc: true
+---
+
+It is crucial to understand where [transaction contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) occurs and how it impacts your workload. Contention is a normal part of database operations. In many cases, it has limited impact on a workload. For example, a large number of contention events with a very short duration may not impact your workload. Similarly, a small number of contention events with longer duration may be acceptable if they occur outside the “user path”. However, if contention substantially contributes to query latency, it should be addressed.
+
+This page shows how to monitor and analyze two types of contention: [lock contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) and [serializable conflicts]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention). These types of contention are exposed via different observability touchpoints:
```

Comment:
it's confusing to me that these two different types of contention link to the same section of the same page

the two links makes me think they are liniking to different things, but they're linking to the same thing

suggest making 'two types of [contention](): ....' the link to that section

Reply:
... update: or the lock contention can link to the troubleshoot lock contention doc